src/vision/pi_vision/pi_face_tracker/nodes/face_tracker.py

```diff
@@ -258,7 +263,10 @@ def __repr__(self):
         )
 
     def update_bounding_box(self):
-        self.bounding_size = self.pt2[1] - self.pt1[1]
+        #is dp 0.22 the correct size for width or height of face?
+        #now using width instead of height as initialized
```

Comment:
Its for the height, not the width, the original formulas were correct.  This can be verified by lloking at https://upload.wikimedia.org/wikipedia/commons/6/61/HeadAnthropometry.JPG measurement number 14, which is 23cm for men, 21 cm for women, in the 50th percentile.  This image is linked from https://en.wikipedia.org/wiki/Human_head

So -- could you change that back, document that it is the height, and give a reference to the above? i.e. add a comment that says this:
```
# According to WP https://en.wikipedia.org/wiki/Human_head the typical height
# of a human face is 22 centimeters. See measurement #14 50th percentile column in
# https://upload.wikimedia.org/wikipedia/commons/6/61/HeadAnthropometry.JPG
```